docs/pkg/models/shared/createpolicyrequest.md

```diff
@@ -9,7 +9,7 @@ The CreatePolicyRequest message is used to create a new policy.
 | ------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------- |
 | `Description`                                                                                                                               | **string*                                                                                                                                   | :heavy_minus_sign:                                                                                                                          | The description of the new policy.                                                                                                          |
 | `DisplayName`                                                                                                                               | **string*                                                                                                                                   | :heavy_minus_sign:                                                                                                                          | The display name of the new policy.                                                                                                         |
-| `PolicySteps`                                                                                                                               | map[string][shared.PolicyStepsInput](../../../pkg/models/shared/policystepsinput.md)                                                        | :heavy_minus_sign:                                                                                                                          | The map of policy type to policy steps. The key is the stringified version of the enum. See other policies for examples.                    |
+| `PolicySteps`                                                                                                                               | map[string][shared.PolicySteps](../../../pkg/models/shared/policysteps.md)                                                                  | :heavy_minus_sign:                                                                                                                          | The map of policy type to policy steps. The key is the stringified version of the enum. See other policies for examples.                    |
```

Comment:
What is the source of these changes? Is this related to the version change?

Reply:
No this is unrelated to the version change, this is just a result of regenerating the SDK. More specifically, when you have an ObjectInput the read only fields are removed. Theres also an ObjectOutput where the readonly fields are added back in.